mtp_noms_ops/apps/security/forms/check.py

```diff
@@ -112,64 +104,43 @@ def clean(self):
         """
         Makes sure that the check is in pending.
         """
+        status = self.cleaned_data['fiu_action']
+        if 'decision_reason' in self.cleaned_data:
+            reason = self.cleaned_data['decision_reason']
+        else:
+            reason = ''
+
+        if reason == '' and status == 'reject':
+            msg = forms.ValidationError('This field is required')
```

Comment:
So on the surface this looks deeply wrong to me, to instantiate a base class of exception outside of the context of a `raise` statement, but it turns out this is explicitly what the django docs recommend https://docs.djangoproject.com/en/2.0/ref/forms/validation/#raising-multiple-errors and this is even the way the `add_error` method works under the hood https://github.com/django/django/blob/master/django/forms/forms.py#L326 Since it's being coerced within `add_error` you technically don't need to instantiate it here, given that we haven't done so elsewhere, but I really don't mind if you want to leave it as is.

Reply:
I think I borrowed this from either somewhere in the code base or django docs, can maybe think about improving later but will ship for now. I can't remember - more reason for more rapid work/deploying :)